src/org/thoughtcrime/securesms/ConversationActivity.java

```diff
@@ -1844,23 +1840,19 @@ private void sendTextMessage(final boolean forceSms, final long expiresIn, final
 
                  final long id = fragment.stageOutgoingMessage(message);
 
-                 Log.w(TAG, "processing ID  " + id);
-                 Log.w(TAG, "processing message ID  " );
                  new AsyncTask<OutgoingTextMessage, Void, Long>() {
                    @Override
                    protected Long doInBackground(OutgoingTextMessage... messages) {
-
                      if (initiatingConversation) {
                        DatabaseFactory.getRecipientDatabase(context).setProfileSharing(recipient, true);
                      }
 
                      String messageBody = messages[0].getMessageBody();
 
-                     if (! (messageBody.length() >= 19 && messageBody.substring(0, 19).equals("{\"type\": \"reaction\""))) {
+                     if (!(messageBody.length() >= 19 && messageBody.substring(0, 19).equals("{\"type\": \"reaction\""))) {
```

Comment:
Don't think we should wait till next sprint for this. I can implement if you want